Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Beam/Booking.hs

```diff
@@ -141,6 +142,7 @@ bookingTMod =
       quoteId = B.fieldNamed "quote_id",
       status = B.fieldNamed "status",
       specialLocationTag = B.fieldNamed "special_location_tag",
+      disabilityTag = B.fieldNamed "person_disability_tag",
```

Comment:
Can you rename field name to `disability_tag` else make it `person_disability_tag` in other tables as well.